RecoEgamma/EgammaElectronAlgos/src/PixelHitMatcher.cc

```diff
@@ -106,52 +114,79 @@ PixelHitMatcher::compatibleSeeds
    typedef std::unordered_map<std::pair<const GeomDet*,GlobalPoint>, TrajectoryStateOnSurface> PosTsosAssoc;
    const int charge = int(fcharge) ;
 
-   const double xmeas_phi = xmeas.phi();
+   // auto xmeas_phi = xmeas.barePhi();
+   auto xmeas_r = xmeas.perp();
+   
+   const float phicut = std::cos(2.5);
```

Comment:
does the 2.5 have any specific meaning?
If it's just a reasonable value magic number, why not use "-0.8" ? It will at least not looks like a constant computed every time.